universal-application-tool-0.0.1/conf/messages

```diff
@@ -150,10 +150,14 @@ link.begin=Start here
 # The text that appears next to an answered question that the applicant can click on to modify the response.
 link.edit=Edit
 
-# Program Prewiew Title
+# An aria-label for screen readers that helps provide context for the associated edit link. The edit link is for a specific
+# question, so this might say "Edit What is your name?". The {0} variable is the question text.
+aria-label.edit=Edit {0}
```

Comment:
(non-blocking) I think `ariaLabel.edit` would be more consistent.

Reply:
good point! will update